.ci/check-for-chart-changes.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
I would opt for `#!/usr/bin/env bash`: https://stackoverflow.com/a/21613044/222044

Reply:
done